accessibility_monitoring_platform/apps/cases/forms.py

```diff
@@ -66,6 +66,7 @@
 DATE_TYPE_CHOICES: List[Tuple[str, str]] = [
     ("audit_case__date_of_test", "Date test started"),
     ("sent_to_enforcement_body_sent_date", "Date sent to EB"),
+    ("case_updated_date", "Case updated"),
```

Comment:
Is this the case updated field in post case summary or the last edited field?

Reply:
It is the field labelled _Case updated_ on the **Post case summary** page.